zebrad/src/components/inbound/tests.rs

```diff
@@ -26,6 +26,9 @@ async fn mempool_requests_for_transactions() {
     let added_transactions = add_some_stuff_to_mempool(&mut mempool_service, network);
     let added_transaction_ids: Vec<UnminedTxId> = added_transactions.iter().map(|t| t.id).collect();
 
+    let mempool_service = BoxService::new(mempool_service);
+    let mempool = ServiceBuilder::new().buffer(20).service(mempool_service);
```

Comment:
The state service works in this same test with a buffer of `1`.

Reply:
Got it. I won't touch this now since you may be working on the file

Reply:
Changed this to `1` in the test at least by now: https://github.com/ZcashFoundation/zebra/pull/2718/commits/fd418b3bfea4a85cac0f0a75ea37ee52c37bcee5